### DISCUSSION TITLE
Can kyuubi operation param "confOverlay" support spark conf

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6286] [Q&A] [answered]
Can param "confOverlay" support spark conf like
`{
                "spark.executor.memory": "2g",
                "spark.driver.memory": "1g",
  }`
https://github.com/apache/kyuubi/pull/4639

**Comment by pan3793** [comment, marked as answer]
Yes, all items in `confOverlay` will be passed to `spark-submit` as `--conf x=y`
